builtin/providers/rancher/config.go

```diff
@@ -3,7 +3,7 @@ package rancher
 import (
 	"log"
 
-	rancherClient "github.com/rancher/go-rancher/client"
+	rancherClient "github.com/rancher/go-rancher/v2"
 	"github.com/raphink/go-rancher/catalog"
```

Comment:
Which part? The `rancherClient` part? I was going for the smallest change possible. And changing every reference didn't seem necessary.